doc/locale_api_WIP.md

```diff
@@ -0,0 +1,46 @@
+
+Note: ListFoo APIs combine the IDs and the translations. 
+If it's easier to return just the IDs and ask for the translations separately,
+let's use such different API.
+
+### Locales
+
+```
+ListLocales(
+  out a(sss) id_english_native
```

Comment:
what langtable has is not locales specific, but language oriented. So here it makes more sense to have something like `out a(sssas) id_english_native_variants` where values will be like `(en,English,English,[en_US.UTF-8,en_GB.UTF-8,en_IN.UTF-8,...]). So in UI user can select language and then variant which is not translated. Similar data we can get for territories where is territory like Zambia and locales and languages speaked there.

Reply:
and when I try to implement it I also found that for some less common languages there are no translations in langtable itself like `Central Aymara`. I expect it will solve when we also get data for filtering to only supported languages